Hast.Abstractions/Hast.Vitis.Abstractions/Readme.md

```diff
@@ -34,6 +34,98 @@ Be sure that all .NET software dependencies are on the same version on both the
+2. Extract it (`tar xzf Xilinx_Unified_2020.2_*`) and copy the folder into the `Docs/Container` folder inside this project.
+3. Download the appropriate CentOS 7 packages from the Vitis Downloads page, or from the Xilinx Lounge if you use Azure.
+4. Extract and copy the RPM files into the `Docs/Container/platform` folder.
+5. Copy the `centos7-install.sh` to the `Docs/Container` as well.
+6. Open a shell of your choice and type `docker build -t vitis .` to create an image. This will take a while.
+7. Open Docker Desktop to verify that the "vitis" image appeared in the *Images* tab.
+8. Clean up after the build is finished with the `docker builder prune -a -f` command.
+9. Go back to Docker Desktop and click *Run* on the "vitis" image.
+10. Expand the *Optional Settings* and create a shared directory by selecting a *Host Path*, and entering "/data" into the *Container Path* field.
+11. Switch to the *Containers / Apps* tab in Docker Desktop and click on the CLI (`>_`) button.
+12. A window with `sh` shell will appear. Type `bash` as it already has the XRT setup configured. 
+13. Copy your Hastlayer project into the shared folder and access it through the `/data` directory.
+    
+As you can see it was as simple as 1, 2, 13!
+
+## Using Azure Attestation
+
+If you want work with an Alveo card on an Azure VM, you need to use the Azure-specific driver. (Currently only `Azure Alveo U250`.) This alters some of the automatic compilation steps and after compilation submits your binary to an attestation server (via Azure Blob Storage) for automatic approval.
+
+### Preparation
+
+You should have received documentation on how to set up the Azure VM, please follow its steps. Once the VM is running, you have to install the correct platform and runtime files from the [Xilinx Lounge](http://www.xilinx.com/member/alveo-platform.html), the Vitis SDK and the .Net runtime or SDK. Transfer all package files into the same directory. Then while in that directory type the following to install them at once:
+
+**Ubuntu**
+```shell
+sudo apt install *.deb
+source ubuntu-install.sh
+```
+
+**CentOS 7**
+```shell
+sudo yum localinstall *.rpm
+source centos7-install.sh
+```
+
+You also need to install .Net 5 and prepare the environment. Source the Ubuntu or CentOS install scripts by typing `source ubuntu-install.sh` or `source centos7-install.sh` respectively. 
+
+
+### Configuration
+
+This feature is currently in the private preview stage. We can't share some specific setting values as they are confidential and you need to set them using information in the validation scripts. If you don't have access to them, please wait until the project leaves the private phase whereupon we will update the fields' default values so you won't have to set them.
+
+The approval process requires addition configuration. Fill out and add the below `AzureAttestationConfiguration` property to the `CustomConfiguration` in your `appsettings.json` file.

```

Comment:
A JSON maradékát is add hozzá, hogy el tudja helyezni az ember, hogy hova rakja (a `HardwareGenerationConfiguration`, `CustomConfiguration` sectionöket).